lib/task_helpers/metrics.rb

```diff
@@ -0,0 +1,41 @@
+module TaskHelpers
+  class Metrics
+    def self.per_tenant
+      data = [%w(tenant_id external_tenant_id number_of_portfolios number_of_products
+                number_of_portfolios_shared number_of_groups_per_portfolios_shared number_of_orders_per_day)]
+      Tenant.all.each do |tenant|
+        ActsAsTenant.with_tenant(tenant) do
+          number_of_days = (DateTime.now.to_date - tenant.created_at.to_date).to_i
+          number_of_orders_per_day_per_tenant = Order.count.to_f / number_of_days.to_f
+
+          data.push [
+            tenant.id,
```

Comment:
@pkomanek For this report the `tenant.id` property is not meaningful, we should drop it.